### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by bhtibrewal**
Added wishlist page 

pages to check 

- [ ] src/pages/wishlist_page/WishlistPage.jsx
- [ ] src/components/card/BasicCard.jsx

### CODE REVIEW

**Review by yaswanthmyneni — APPROVED** [review]
you can merge.

**Review by sri-rishi — APPROVED** [review]
Code looks great, feel free to merge

File: src/components/card/BasicCard.jsx [hunk 1]
```diff
@@ -1,35 +1,40 @@
...
+        <i className="fa-regular fa-heart fa-2x"></i>
       </button>
-      <div class="content">
+      <div className="content">
         <img
-          class="card-img"
+          className="card-img"
           src="https://render.fineartamerica.com/images/images-new-artwork/images-medium-5/la-porta-rossa-sulla-salita-guido-borelli.jpg?v=2"
           alt=""
         />
```
Comment (Ankur9669) [inline src/components/card/BasicCard.jsx:22]: Alt tag😂

File: src/pages/wishlist_page/WishlistPage.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+export const WishlistPage = () => {
+  return (
+    <main className="main">
+      <section className="page-header-section">
+        <div className="section-bg">
+          <img
+            src="https://d17h7hjnfv5s46.cloudfront.net/assets/build/images/banner_wishlist_no_connect.cbbca931.jpg"
+            alt=""
+          />
```
Comment (Ankur9669) [inline src/pages/wishlist_page/WishlistPage.jsx:22]: Suggestion: Alt tag

File: src/pages/wishlist_page/WishlistPage.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        </div>
+      </section>
+
+      <section className="wishlist-sec">
+        {Array(6)
+          .fill()
+          .map(() => {
+            return (
+              <BasicCard item={item}>
+                <ButtonPrimary>
```
Comment (Ankur9669) [inline src/pages/wishlist_page/WishlistPage.jsx:36]: Suggestion: Use key attribute for optimisation

**Review by Ankur9669 — APPROVED** [review]
The code looks fine no major changes required looks good to merge

File: src/pages/wishlist_page/WishlistPage.jsx (hunk 2 above)
Comment (bhtibrewal) [reply src/pages/wishlist_page/WishlistPage.jsx:22]: please look into it again it has alt tag @Ankur9669

File: src/components/card/BasicCard.jsx (hunk 1 above)
Comment (bhtibrewal) [reply src/components/card/BasicCard.jsx:22]: please look into it again it has alt tag @Ankur9669

**Pull request merged**